worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -129,8 +129,8 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
     if (!hdfsDir.isEmpty) {
       val path = new Path(hdfsDir)
       val scheme = path.toUri.getScheme
-      val disableCacheName = String.format("fs.%s.impl.disable.cache", scheme);
-      val hdfsConfiguration = new Configuration
+      val disableCacheName = String.format("fs.%s.impl.disable.cache", scheme)
+      val hdfsConfiguration = CelebornHadoopUtils.newConfiguration(conf)
```

Comment:
seems `spark.hadoop.*` does not take effect?

Reply:
> seems `spark.hadoop.*` does not take effect?

Yea, to support this is complex, and not necessary, also we only support `spark.celeborn.hadoop.xx.xx` can separate conf cover Hadoop configuration's working scope.

Reply:
I'm fine with this. cc @FMX too, seems no HDFS docs now, this should be documented clearly, otherwise may confusing Spark users, especially who run Spark on K8s, because there were some issues on Hadoop Conf propagation in Spark on K8s, the user is likely configuration Hadoop through `spark.hadoop.*` in `spark-defaults.conf` instead